src/main/java/hudson/plugins/active_directory/ActiveDirectoryDomain.java

```diff
@@ -266,6 +272,15 @@ public ListBoxModel doFillTlsConfigurationItems() {
             return model;
         }
 
+        @RequirePOST
+        public FormValidation doCheckTlsConfiguration(@QueryParameter String tlsConfiguration) {
+            Jenkins.get().checkPermission(Jenkins.MANAGE);
```

Comment:
CI is failing because MANAGE is not allowed in this plugin. There're 2 options here:
* add `<useBeta>true</useBeta>` to the properties in pom.xml
* Change Jenkins.MANAGE into Jenkins.ADMINISTER

Reply:
yes, I have done that locally, but I forgot to push :(